Libraries/test/TestServerlessApp/SimpleCalculator.cs

```diff
@@ -8,7 +8,7 @@
 using TestServerlessApp.Services;
 
 // Assembly attribute to enable the Lambda function's JSON input to be converted into a .NET class.
-[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
+//[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
```

Comment:
can this be removed completely? (even the comment)

Reply:
I have uncommented the line as it is required.